lib/identity_cache/belongs_to_caching.rb

```diff
@@ -51,5 +52,15 @@ def #{options[:prepopulate_method_name]}(record)
         CODE
       end
     end
+
+    def load_and_readonlyify(association_name)
```

Comment:
This is also used in the QueryApi, which shouldn't depend on the BelongsToCache module.  The QueryApi module would be a more suitable place for this method.